src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -711,7 +711,7 @@ public override bool Equals([NotNullWhen(true)] object? comparand)
 
         public override int GetHashCode()
         {
-            return (Name + "=" + Value + ";" + Path + "; " + Domain + "; " + Version).GetHashCode();
+            return HashCode.Combine(Name, Value, Path, Domain, Version);
```

Comment:
```suggestion
            return HashCode.Combine(StringComparer.OrdinalIgnoreCase.GetHashCode(Name), Value, Path, StringComparer.OrdinalIgnoreCase.GetHashCode(Domain), Version);
```
I realize it's an existing issue, but this one doesn't match the `Equals`